src/components/Common/PaginationControl/PaginationControlTypes.ts

```diff
@@ -6,6 +6,6 @@ export type PaginationControlProps = {
   handleItemsPerPageChange: (n: number) => void
   currentPage: number
   totalPages: number
+  itemsPerPage: string
```

Comment:
I realized since the Select instance in PaginationControl is constrained with values, we likely need to make this an enum prop. We could do

```suggestion
  itemsPerPage: 5 | 10 | 50
```

Reply:
makes sense